javascript/javascript1/week4/aliSharifi/vowelCount.js

```diff
@@ -0,0 +1,34 @@
+// 6kyu Vowel Count
+function encode(string) {
+    return string.replace(/[aeiou]/g, char => ({ 'a': '1', 'e': '2', 'i': '3', 'o': '4', 'u': '5' }[char]));
+}
+
+function decode(string) {
+    return string.replace(/[1-5]/g, num => ({ '1': 'a', '2': 'e', '3': 'i', '4': 'o', '5': 'u' }[num]));
+}
+
+console.log(encode("HYF is the Best")); 
```

Comment:
This code is nice, but Kata's instruction was to return the number (count) of vowels in the given string. So it would look like:
```js
function getVowelsCount(string) {
  // Your code here
}

console.log(getVowelsCount("HYF is the Best")); // Returns 3 
```